test/sensor_simulator/flow.cpp

```diff
@@ -29,7 +28,7 @@ void Flow::setData(const flowSample& flow)
 flowSample Flow::dataAtRest()
 {
 	flowSample _flow_at_rest;
-	_flow_at_rest.dt = 0.02f;
+	_flow_at_rest.dt = static_cast<float>(_update_period) * 1e-6f;
```

Comment:
@kamilritz Would it be fine that way? So the user can get the `dataAtRest`, modify `dt` if he wants (generate errors), but the default value is always correct

Reply:
yes, that is a good option